docs/USAGE.md

```diff
@@ -2,37 +2,95 @@
 
 There are several steps that must be performed in order to use OVE to control a display.
 
-## Setting up OVE
-
 Before using OVE, you will need to install [**OVE Core**](https://github.com/ove/ove), any [**OVE Apps**](https://github.com/ove/ove-apps), and any [**OVE Services**](https://github.com/ove/ove-services) that you intend to use. Installation guidelines can be found in the [OVE Installation Guide](./INSTALLATION.md).
 
-You will also need to modify the [``ove/packages/ove-core/src/client/Spaces.json``](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file to define a new ``space`` in OVE core.
+## Setting up OVE
 
-Note that all services need to hosted somewhere that is accessible from the computers connected to the monitors that will be used in the display.
+By default, OVE provides two `spaces`, that are useful for trying out some of the [**OVE Apps**](https://github.com/ove/ove-apps) with sample content. These are:
 
-## Creating content
+* `LocalNine` - This `space` contains nine `clients` arranged in `3 x 3` configuration. Each `client` has a dimension of `1440 x 808` pixels and the total `space` has a dimension of `4320 x 2424` pixels.
+* `LocalFour` - This `space` contains four `clients` arranged in `2 x 2` configuration. Each `client` has a dimension of `1440 x 808` pixels and the total `space` has a dimension of `2880 x 1616` pixels.
 
-When designing content to be displayed in an OVE environment, be aware of the [potential pitfalls](./PITFALLS.md).
+You will need to modify the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file and introduce a new `space` in OVE, before it can be used in a production setting.
```

Comment:
```suggestion
You will need to modify the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file and introduce a new `space` in OVE, before it can be used.
```

If the space isn't defined it can't be used *at all*, whether for testing/experimentation or in production.